src/core/migrations/0083_auto_20230627_1400.py

```diff
@@ -0,0 +1,39 @@
+"""Set NULL Account names (first, middle, last) to the empty string.
+
+As per django recomendation:
+https://docs.djangoproject.com/en/4.2/ref/models/fields/#null
+
+Also solves the problem of people who want to sign their papers with
+their "art" names (e.g. crossref deposits with empty <given_names>.
+
+"""
+# Generated by Django 3.2.19 on 2023-06-27 12:00
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def set_null_names_to_empty_string(apps, schema_editor):
+    """Set NULL Account names (first, middle, last) to the empty string."""
+    Account = apps.get_model("core", "Account")  # noqa N806
+    for account in Account.objects.filter(
+        Q(first_name=None) | Q(middle_name=None) | Q(last_name=None),
+    ):
+        if account.first_name is None:
+            account.first_name = ""
+        if account.middle_name is None:
+            account.middle_name = ""
+        if account.last_name is None:
+            account.last_name = ""
+        account.save()
```

Comment:
When autogenerating a migration that makes a field non-nullable, you can provide a default value to the `makemigrations` command, so this shouldn't be necessary